src/typegate/engine/runtime.d.ts

```diff
@@ -318,7 +318,12 @@ export type OperationEvent =
   | { type: "Send"; event_name: string; value: unknown }
   | { type: "Stop"; result: unknown }
   | { type: "Start"; kwargs: Record<string, unknown> }
-  | { type: "Compensate" };
+  | {
+    type: "Compensate";
+    save_id: number;
+    error: string;
+    compensation_result: any;
+  };
```

Comment:
Underlying porotbuff type is not generated and converted into serde type.
You may want to look at [gen-subs-protoc](https://github.com/metatypedev/metatype/blob/main/tools/tasks/gen.ts#L17).